e2e/deployers/disapp.go

```diff
@@ -99,6 +100,18 @@ func (d DiscoveredApp) Undeploy(ctx types.TestContext) error {
 		return err
 	}
 
+	deadline := time.Now().Add(util.UndeployTimeout)
```

Comment:
If we rebase on #2043 we don't need to create the deadline here, it is part of the context.